feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -155,6 +155,9 @@ $sapFioriContent_LabelColor: #6a6d70;
       padding-bottom: 0;
     }
     max-height: calc(100vh - 260px);
```

Comment:
This height is too tall for mobile. With this height, the header including 'create new customer' button will be placed outside the visible range.